src/main/java/io/khasang/bend/config/DataConfig.java

```diff
@@ -38,7 +38,6 @@ public JdbcTemplate jdbcTemplate() {
         return jdbcTemplate;
     }
 
-
```

Comment:
Why this class at pull request?

Reply:
accidentally added, just formatting everywhere)